.github/workflows/ecosystem-ci-trigger.yml

```diff
@@ -22,57 +66,147 @@ jobs:
                 repo: context.repo.repo,
                 username: user,
               });
-              hasTriagePermission = data.user.permissions.triage
+              hasTriagePermission = ['triage', 'write', 'admin'].some(p => data.user.permissions[p]);
```

Comment:
I guess this diff is not needed. `permissions.triage` is always true for the ones that has `permissions.write`/`permissions.admin`.

Reply:
done!